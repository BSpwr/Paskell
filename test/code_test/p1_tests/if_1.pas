program If1Test;
var
   grade: string = 'A';
   good: boolean = true;
   junk, junk2: string;
   val: real = 5.0;
   b: real = 5.0;
   s, a: real;
begin

    grade := 'B';

   if grade = 'B' then
      begin
      writeln('good');
      writeln('second statement of compound')
      end
   else if grade = 'A' then
      writeln('WOW A!')
   else if grade <> 'F' then
      writeln('Not Fail!')
   else writeln('Bad job, its an F');
   
   val := 7;
   b := 7;
   writeln(val + ((b+1) * 2));
   writeln('Your grade is an: ', grade);

end.