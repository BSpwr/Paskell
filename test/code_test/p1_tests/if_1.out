good
second statement of compound
23.0
Your grade is an: B
